### DISCUSSION TITLE
Add commit lint husky hook

### DISCUSSION THREAD

**Discussion by somratAtSekaiLab** [monstar-lab-oss/admin-panel-template-reactjs#129] [General]
Commit messages should follow the [Conventional Commits](https://www.conventionalcommits.org/) rule followed by [Commit lint config conventional](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional)

Useful links: 
https://github.com/conventional-changelog/commitlint

**Comment by jacksonjp** [comment]
As discussed, we are squashing and merging PR's which goes as a single commit. Hence i guess it makes sense to install some app like mergeable.
